api/lib/firewall_functions.pl

```diff
@@ -324,13 +327,38 @@ sub list_fw_rules
         $props{'Time'} = get_time_info($props{'Time'}, $fw->{'ftdb'}, $expand);
         $props{'Service'} = get_service_info($props{'Service'}, $fw, $expand);
         $props{'Position'} = int($props{'Position'});
-        $max_pos = max($max_pos, $props{'Position'});
+        $max_posRule = max($max_posRule, $props{'Position'});
         $nextID =`/usr/libexec/nethserver/api/nethserver-firewall-base/lib/rules-next-id`;
         push(@rules, \%props);
         $i++;
     }
+    foreach ($fw->getSeparator()) {
+        my %props = $_->props;
+        if ($skip_local) {
+            # skip rule to/from the firewall
+            next if ($props{'Dst'} eq 'fw');
+        } else {
+            # skip rule not for the firewall itself
+            next if ($props{'Dst'} ne 'fw');
+        }
```

Comment:
We do not need this if we drop the `Dst` field.

Reply:
done